packages/commonwealth/client/scripts/views/pages/NotificationSettings/SubscriptionEntry.tsx

```diff
@@ -64,7 +64,7 @@ export const SubscriptionEntry = (props: SubscriptionEntryProps) => {
         <div>
           <CWCommunityAvatar
             community={{
-              iconUrl: thread.Community.icon_url,
+              iconUrl: thread.Community.icon_url!,
```

Comment:
Shouldn't this be `thread.Community?.icon_url` i.e. `ChainInfo.iconUrl` should be optional given that it is not guaranteed to exist?

Reply:
I don't know honestly.  I think what's happening is that our default dataset has a null icon URL. Maybe we need to change that but for some reason it immediately caused problems for me.

I don't think in practice new data will have this issue because we have safeguards for it.  

The problem is that now we're in a weird situation with nulls because StrictNullChecks weren't on so now we have a lot of code that doesn't know about nulls without an elegant way to handle it.

I think practically we have to use "!" or change the types and then manually find any bugs.